Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -27,27 +26,38 @@ class DisplayHymnViewModel: ObservableObject {
     }
 
     func fetchHymn() {
-        repository
-            .getHymn(hymnIdentifier: identifier)
-            .map({ (hymn) -> Title? in
-                guard let hymn = hymn else {
-                    return nil
-                }
-
-                if self.identifier.hymnType == .classic {
-                    return "Hymn \(self.identifier.hymnNumber)"
-                }
-                let title = hymn.title.replacingOccurrences(of: "Hymn: ", with: "")
-                return title
-            })
-            .receive(on: mainQueue)
-            .sink(
-                receiveValue: { [weak self] title in
-                    self?.title = title ?? ""
-            }).store(in: &disposables)
+        favoritedStatus = RealmHelper.checkIfFavorite(identifier: self.identifier)
+        if favoritedStatus == true {
+            self.title = RealmHelper.retrieveTitle(identifier: self.identifier)
+            return
+        } else {
```

Comment:
I don't think we need to do this. We should always get the hymn from the repo. In the future, we're going to need to get the music, the sheet music, the related songs, languages, etc, so we'll need to do this regardless. the title is really just for displaying a list of favorited songs

Reply:
Okay, taking it out